corehq/pillows/sofabed.py

```diff
@@ -22,8 +23,12 @@ def process_sql(self, doc_dict, delete=False):
                 pass
             return
 
+        if doc_dict.get('xmlns') == DEVICE_LOG_XMLNS:
```

Comment:
oh wait, if you ever reindex, you'll run into old docs that are device logs. :+1: